### PR TITLE
[jenkins] Fix of building images in PR job

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lukywill16@gmail.com>

### Type of change

- Bugfix

### Description

This PR fixes building images in our PR jobs, which failed because of missing `shellcheck`.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

File: .jenkins/jenkins.groovy [hunk 1]
```diff
@@ -7,6 +7,14 @@ def setupKubernetes() {
     sh(script: "${workspace}/.azure/scripts/setup-kubernetes.sh")
 }
 
+def setupShellheck() {
+    // set SElinux to permisive mode
+    sh(script: "sudo setenforce 0")
```
Comment (Frawless) [inline .jenkins/jenkins.groovy:12]: isn't this already used somewhere in the pipeline?

**Review by Frawless — APPROVED** [review]

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Comment by im-konge** [comment]
FYI - PR job fails on pushing images to local registry, which is not correctly set up -> because of `ImagePullBackOff` and due reaching the pull limit

**Review by scholzj — APPROVED** [review]

**Review by sknot-rh — APPROVED** [review]

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Comment by im-konge** [comment]
@strimzi-ci run tests profile=regression testcase=ConnectBuilderST

**Pull request merged**